QUARKUS-5623.md

```diff
@@ -0,0 +1,36 @@
+# QUARKUS-5623 - Logging in OpenTelemetry
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-5623
+
+Upstream PRs:
+    - https://github.com/quarkusio/quarkus/pull/38239
+    - https://github.com/quarkusio/quarkus/pull/46354
+
+Documentation: https://quarkus.io/version/main/guides/opentelemetry-logging
+
+## Preview status
+OpenTelemetry logging is currently considered dev-preview in quarkus.
+
+## Scope of testing
+Test OpenTelemetry quarkus extension.
+Verify that quarkus application is sending its logs to the Grafana backend.
+Verify that log messages as well as log levels are correct.
```

Comment:
Bullet-points would be better for this part

Reply:
Changed.